### PR TITLE
Part 2

### PR DESCRIPTION
**Pull request by elabcius5**
For this PR, I created a client for consuming the service from Octokit API, I associated a model acording to JSON and then I consumed the service in Activity. For now, I log the data in console debug

### CODE REVIEW

File: app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
+package com.globant.tutoria.subscriberssearch.model;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Created by juan.herrera@globant.com on 11/08/2016.
+ */
+public class SubscribersModel {
+
+    @SerializedName("login")
```
Comment (lucaslabs) [inline app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java]: No need to use `SerializedName` when the name on the JSON field is the same of the attribute.

File: app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    private String login;
+    @SerializedName("id")
+    private String id;
+    @SerializedName("avatar_url")
```
Comment (lucaslabs) [inline app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java:12]: This case is where `SerializedName` is valid.

File: app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java [hunk 3]
```diff
@@ -0,0 +1,180 @@
+package com.globant.tutoria.subscriberssearch.model;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Created by juan.herrera@globant.com on 11/08/2016.
+ */
+public class SubscribersModel {
```
Comment (lucaslabs) [inline app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java]: This model can be immutable: final class, final attributes, NO setters.

**Comment by lucaslabs** [comment]
@elabcius5 please add a video that showcase the app running with this changes :smiley:

**Comment by elabcius5** [comment]
The video was upload on YouTube. The url is: https://www.youtube.com/watch?v=ASWHeyzsVgM

File: app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java [hunk 4]
```diff
@@ -5,176 +5,128 @@
...
  * Created by juan.herrera@globant.com on 11/08/2016.
  */
-public class SubscribersModel {
+public final class SubscribersModel {
 
-    @SerializedName("login")
-    private String login;
-    @SerializedName("id")
-    private String id;
+    final String login;
```
Comment (lucaslabs) [inline app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java]: Private access modifier must be here, this is, `private final String login`. Do the same for the rest of the fields/attributes.

**Comment by juliocatano** [comment]
please merge master into this branch, since your first PR was already merged because I continue to see the change for the .gitignore

**Comment by juliocatano** [comment]
**LG** after small comment

**Pull request merged**